testing/ansible-tester/Dockerfile

```diff
@@ -1,29 +1,69 @@
-FROM williamyeh/ansible:centos7
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
 
 ### Update system packages
-RUN yum --assumeyes --quiet upgrade
+RUN apt-get -q -y update
+RUN apt-get -q -y upgrade
+
+RUN apt-get install -q -y \
+    dmidecode \
+    python3 \
+    python3-pip \
+    gpg-agent \
+    rpm  \
+    openssh-client \
+    wget
 
-### Install additional ansible support
-RUN yum --assumeyes --quiet install dmidecode python3 rpm-build wget
 # XXX - The latest version of pip (21.3.1) is breaking on CentOS 7 without
 #       following encoding setup.
-ENV LANG=en_US.UTF-8
+ ENV LANG=en_US.UTF-8
 # XXX - ^^^
+
+# XXX - To resolve ansible.utils.display.initialize_locale (unsupported locale setting).
+ ENV LC_ALL=C
+ # XXX - ^^^
+
 RUN python3 -m pip --quiet install --upgrade pip
 RUN python3 -m pip --disable-pip-version-check --quiet install \
-  dnspython netaddr python-irodsclient requests urllib3 wheel
-# NB: ansible-core 2.12+ requires Python 3.8, which can't be installed from the
-#     epel on CentOS 7.
-RUN python3 -m pip --disable-pip-version-check --quiet install 'ansible-core<2.12'
+  dnspython netaddr python-irodsclient requests urllib3 wheel jinja2 ansible-core==2.11.12
 
 ### Install inspection support
-RUN yum --assumeyes install ca-certificates epel-release
-RUN python -m pip --quiet uninstall --yes urllib3
-
-ADD https://packages.irods.org/renci-irods.yum.repo /etc/yum.repos.d/renci-irods.yum.repo
+RUN apt-get install -y ca-certificates
+
+RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add -
+RUN echo "deb [arch=amd64] https://packages.irods.org/apt/ bionic main" | tee /etc/apt/sources.list.d/renci-irods.list
```

Comment:
Since there's no need to elevate permissions to create `renci-irods.list`, i.e., `sudo` isn't needed the `echo` can be redirected to the file instead of piping through `tee`. I.e., `RUN echo "..." > /etc/.../renci-irods.list`.